joomla/joomla-cms
# Module Positions not show in com_user settings - Fix for issue #38705

Write the field type different to fix issue https://github.com/joomla/joomla-cms/issues/38705

Pull Request for Issue #38705 .

### Summary of Changes
Changed the field Name, maybe because of Case Sensitive Servers?


### Testing Instructions
If you experience that you have no Module Position selectable in the com_user Settings for Multifactor authentification like descriped here: https://github.com/joomla/joomla-cms/issues/38705
Apply the patch and see if the module Positions are listed now.


### Actual result BEFORE applying this Pull Request
No module positions are shown

### Expected result AFTER applying this Pull Request
Module Positions are shown.


### Link to documentations
Please select:
